scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -80,108 +79,112 @@
 y.value_counts(True)
```

Comment:
@rdvelazquez perhaps you have commits pending as part of a review? Refresh page?